packages/main/src/types/TableCellAlign.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Alignment of the &lt;ui5-table-cell&gt; component.
+ *
+ * @public
+ */
+enum TableCellAlign {
+    /**
+     * @public
+     */
+    Left = "left",
+
+    /**
+     * @public
+     */
+    Begin = "start",
```

Comment:
Not sure if `Begin = "start"` works here. What comes up in the documentation as a value for this enum? Is it Begin or start?

That is pretty crucial. If the doc is showing Begin as value to fill in here, it won't work, because the CSS variable's value will be "Begin" instead of the expected "start", because the value is taken as a pure string.

Reply:
We can also think of changing Begin to Start

@aborjinik opinions?